components/browser/engine-system/src/main/java/mozilla/components/browser/engine/system/SystemEngineSession.kt

```diff
@@ -231,6 +231,8 @@ class SystemEngineSession(
 
             // webViewSettings built-in zoom controls are the only supported ones, so they should be turned on.
             webSettings.builtInZoomControls = true
+            // Disable zoom button on pages #1628
+            webView.settings.displayZoomControls = false
```

Comment:
When you're in this code block, you can use `webSettings` to change the `displayZoomControls`.

So it would just be `webSettings.displayZoomControls`.